client/src/components/CardList/CardList.tsx

```diff
@@ -2,12 +2,12 @@ import _ from "lodash";
 import React from "react";
 
 interface CardListElementChildProps {
-  display: string | React.ReactNode;
-  href: string;
+  display: React.ReactNode;
+  href?: string;
 }
 
 interface CardListElementProps {
-  display: string | React.ReactNode;
+  display: React.ReactNode;
   href: string;
```

Comment:
Haven't looked in the render, but I think this top-level `href` prop should also be optional? Double check, might have to update the story too depending.